### PR TITLE
Dz1

### PR DESCRIPTION
**Pull request by VasilevichDev**
DZ 1

### CODE REVIEW

**Review by YudjinFoggy — APPROVED** [review]

**Review by YarikGuk — APPROVED** [review]

File: app/src/main/java/com/example/homework/PictureActivity.kt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_picture)
+
+        val url = findViewById<EditText>(R.id.editText)
+        val textUrl = url.text.toString()
+        val picture = findViewById<ImageView>(R.id.imageView)
+        val button = findViewById<Button>(R.id.Button)
+
+        button.setOnClickListener {
+            Picasso.get().load(textUrl).into(picture)
```
Comment (Berikz80) [inline app/src/main/java/com/example/homework/PictureActivity.kt]: exception if you get empty textUrl

**Review by Berikz80 — CHANGES_REQUESTED** [review]
make check for empty string textUrl

File: app/src/main/java/com/example/homework/PictureActivity.kt (hunk 1 above)
Comment (VasilevichDev) [reply app/src/main/java/com/example/homework/PictureActivity.kt]: fixed)

**Review by Berikz80 — APPROVED** [review]
good job

**Review by e-kleban — APPROVED** [review]

**Review by Gizcerbes — APPROVED** [review]

**Review by TMSlebedev — APPROVED** [review]

**Pull request merged**